ext/standard/basic_functions.stub.php

```diff
@@ -464,8 +464,7 @@ function forward_static_call(callable $callback, mixed ...$args): mixed {}
 
 function forward_static_call_array(callable $callback, array $args): mixed {}
 
-/** @return false|null */
-function register_shutdown_function(callable $callback, mixed ...$args): ?bool {}
+function register_shutdown_function(callable $callback, mixed ...$args): void {}
```

Comment:
This change should be ported back to PHP 8.1, shouldn't it?

Reply:
Is it harmful if it isn't? Might just have some wrong info for the optimizer but that's not the most dramatic?

Reply:
Yeah, it's a bit harmful for optimizer for sure, but I don't know how dramatic the effects are.